www/js/control/ProfileSettings.jsx

```diff
@@ -24,39 +27,20 @@ const ProfileSettings = () => {
     const { appConfig, loading } = useAppConfig();
     const { colors } = useTheme();
 
-    // get the scope of the general-settings.js file
-    const mainControlEl = document.getElementById('main-control');
-    const settingsScope = angular.element(mainControlEl.querySelector('profile-settings')).scope();
-    console.log("settings scope", settingsScope);
-    
-    // grab any variables or functions we need from it like this:
-    const { showLog, showSensed } = settingsScope;
-
     //angular services needed
     const CarbonDatasetHelper = getAngularService('CarbonDatasetHelper');
     const UploadHelper = getAngularService('UploadHelper');
     const EmailHelper = getAngularService('EmailHelper');
-    const ControlCollectionHelper = getAngularService('ControlCollectionHelper');
-    const ControlSyncHelper = getAngularService('ControlSyncHelper');
     const KVStore = getAngularService('KVStore');
     const NotificationScheduler = getAngularService('NotificationScheduler');
     const ControlHelper = getAngularService('ControlHelper');
     const ClientStats = getAngularService('ClientStats');
     const StartPrefs = getAngularService('StartPrefs');
     const DynamicConfig = getAngularService('DynamicConfig');
 
-    if (!controlUpdateCompleteListenerRegistered) {
-        settingsScope.$on('control.update.complete', function() {
-            console.debug("Received control.update.complete event, refreshing screen");
-            refreshScreen();
-            refreshCollectSettings();
-        });
-        controlUpdateCompleteListenerRegistered = true;
-    }
-
     //functions that come directly from an Angular service
```

Comment:
This comment is no longer true?!

Reply:
I updated it, thanks for pointing that out!